encoding/wkt/wkt_encode_test.go

```diff
@@ -0,0 +1 @@
+package wkt
```

Comment:
should this be under the namespace wkt_test or should the file be named with an _internal suffix?